src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Win32.cs

```diff
@@ -259,56 +256,51 @@ internal static ProcessInfo[] GetProcessInfos(Predicate<int>? processIdFilter =
             // Start with the default buffer size.
             int bufferSize = DefaultCachedBufferSize;
 
-            // Get the cached buffer.
-            long[]? buffer = Interlocked.Exchange(ref CachedBuffer, null);
-
-            try
+            while (true)
             {
-                while (true)
-                {
-                    if (buffer == null)
-                    {
-                        // Allocate buffer of longs since some platforms require the buffer to be 64-bit aligned.
-                        buffer = new long[(bufferSize + 7) / 8];
```

Comment:
That _7/8_ magic number is gone now 🙀

Reply:
Should we backport this change to 1.0? 🤣 

Jokes aside, most probably this magic was forcing the array to be aligned?